frontend/src/screens/learner.tsx

```diff
@@ -64,33 +66,74 @@ const LearnerDashboard = () => {
             <HighlightedStudies />
 
             <StudiesContainer />
-
+            <Affix position={{ bottom: 20, left: 20 }}>
+                <Transition transition="slide-up" mounted={isVisible}>
+                    {(transitionStyles) => (
+                        <Button
+                            onClick={() => scrollTo({ y: 0 })}
+                            style={transitionStyles}
+                            radius="xl"
+                            p={0}
+                            w={48}
+                            h={48}
+                            bg="#007bff"
+                        >
+                            <IconArrowUp size="1.5rem" color="#fff" />
+                        </Button>
+                    )}
+                </Transition>
+            </Affix>
             <Footer includeFunders />
         </div>
     )
 }
 
-export const SearchBar: FC<{search: string, setSearch: (search: string) => void}> = ({ search, setSearch }) => {
-    const isMobile = useIsMobileDevice()
+export const SearchBar: FC<{
+    search: string;
+    setSearch: (search: string) => void;
+}> = ({ search, setSearch }) => {
+    const isMobile = useIsMobileDevice();
 
     return (
         <TextInput
-            w={isMobile ? '100%' : '400px'}
-            size='lg'
+            w={isMobile ? '100%' : '600px'}
+            size='md'
             value={search}
             onChange={(event) => setSearch(event.currentTarget.value)}
-            rightSection={search.length ?
-                <IconX onClick={() => setSearch('')} style={{ cursor: 'pointer' }}/> :
-                <IconSearch />
+            rightSection={
+                search.length ? (
+                    <IconX onClick={() => setSearch('')} style={{ cursor: 'pointer' }}/>
+                ) : (
+                    <ActionIcon variant="subtle" color="#848484">
```

Comment:
Use the colors from theme, in this case it is colors.text

Reply:
@divitcr7 did you see this comment?